testing/integration-neo4j/src/test/java/com/neueda/jetbrains/plugin/graphdb/test/integration/neo4j/tests/cypher/parameters/CypherParametersProviderTest.java

```diff
@@ -8,21 +8,30 @@
 
 import java.util.Map;
 
-import static org.assertj.core.api.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class CypherParametersProviderTest extends BaseIntegrationTest {
 
     private class TestParametersProvider implements ParametersProvider {
 
-        private String parametersJson;
+        private String globalParametersJson, fileSpecificParametersJson;
```

Comment:
Make 2 separate fields.

Reply:
Fixed in [b481123](https://github.com/neueda/jetbrains-plugin-graph-database-support/pull/107/commits/b481123a4b4a9620b825b946d8ac783aa1d0c49f)